### DISCUSSION TITLE
Add settings for a repo

### DISCUSSION THREAD

**Discussion by biel-correa** [community/community#16452] [Mobile] [answered]
I can't view all tabs that show on repo settings in the desktop version

**Comment by mxie** [comment, marked as answer]
Hi @biel-correa,

Can you clarify what you're looking for? Are you looking for a Settings tab or some other tab that's missing?

**Reply by biel-correa** [reply]
Hey @mxie,

I'm looking for a settings tab like in the desktop version.

**Reply by mxie** [reply]
Got it - thanks for clarifying! Unfortunately, this is currently not on our roadmap. We'll reconsider it if there's enough interest though.

Can you tell us more about what you'd like to be able to accomplish by having Settings available?

**Reply by ErSpieltHD** [reply]
> Can you tell us more about what you'd like to be able to accomplish by having Settings available? @mxie

Like deleting the repo, making it public/private. Add webhooks.
Literally everything you do on desktop.

The mobile app should be a near 1:1 clone of the desktop app imo.

**Reply by ghost** [reply]
> > Can you tell us more about what you'd like to be able to accomplish by having Settings available? @mxie
> 
> Like deleting the repo, making it public/private. Add webhooks.
> Literally everything you do on desktop.
> 
> The mobile app should be a near 1:1 clone of the desktop app imo.

**Comment by AlbertRamstein** [comment]
> I can't view all tabs that show on repo settings in the desktop version 

-if you activate setting.ijon maybe the desktopvideo version will react on the operative repo system...

**Reply by khaleefa222** [reply]
...

**Comment by DarkDose75** [comment]
f6b06c7b8cd7ab004c0ff96cdafeff9da806fbea ▲◄►◄{_www.web-links_}/ ► ◄►▲]

**Comment by azammana** [comment]
> I can't view all tabs that show on repo settings in the desktop version 

Hi brother I need to your help

**Comment by azammana** [comment]
> I can't view all tabs that show on repo settings in the desktop version 

Hi brother I need to your help
